### DISCUSSION TITLE
Standardize deployment of login infos (port, username, password, IP)

### DISCUSSION THREAD

**Discussion by chris400** [community-scripts/ProxmoxVE#131] [Ideas]
Currently some scripts provide information about the needed login info on the community scripts's website, some display it just once after deployment, some do not provide such thing.

The script of paperless-ngx does it pretty well: It creates a file /root/paperless.creds which contains most needed info.
I would suggest to apply this to all scripts and go a little further: publish these infos to the container's notes.

**Comment by Mellowlynx** [comment]
It depends on how the app handles the credentials.
Some come with non, (aka first user that registrars will be an admin), some come with defaults, and others are set up during install.
So standardization is probably not possible.

**Reply by chris400** [reply]
May I suggest to exactly put that into the container's notes?
e.g.

- credentials are: abcde _OR_
- credentials will be created on first login

**Reply by Mellowlynx** [reply]
If credentials are pre setup, it's included on the site:
![image](https://github.com/user-attachments/assets/73d7528c-2189-454b-9c06-61dde73c519c)

If they are made during install, they are shown at the end.

At this point, I do not know for sure If there is an app added that will admin the first made user, but I know they exist.

Adding it to the notes within Proxmox, would require significant changes, this will probably not happen.

**Reply by darc432** [reply]
I think this a very good idea, as most of the time is hard for me to remember the IP and PORT or PATH, where the username and password I will always remember. E.g. For pi-hole if I type only the IP address it will show me the Lighttpd placeholder page, where if I had in the notes that I have to go to "ip.add.re.ss/admin" to reach the interface I will know right away.
Another problem is to find the IP address of the LXC, for this I have to go into LXC's console and type "ip a". If we can make it in such a way where the notes are updated dinamically to the current IP address and just show the proper URL (e.g. 192.168.10.10/admin). Makeing it clickable it will be extraordinary.  
The above is very similar to Unraid aproach for dockers where you can right click and reach the WebUI. See below an example

![image](https://github.com/user-attachments/assets/48fdc7d8-4b31-4e00-a5b1-7d00bbf274f0)

Another model from casaos
![image](https://github.com/user-attachments/assets/641c2e31-ec14-4b8e-ae55-33e27bf31e29)

**Reply by darc432** [reply]
I found this lxc-iptag that is getting and setting the IP as a tag for LXC. It can be used as a reference for this functionality.
https://github.com/gitsang/lxc-iptag/blob/main/README.md 

This should be added also as a separate script for Proxmox VE Tools (MISC), as is very useful.

**Reply by MickLesk** [reply]
For this is already an active PR
